packages/crashlytics/ios/RNFBCrashlytics/RNFBCrashlyticsModule.m

```diff
@@ -101,31 +100,7 @@ + (BOOL)requiresMainQueueSetup {
       rejecter:
       (RCTPromiseRejectBlock) reject) {
   if ([RNFBCrashlyticsInitProvider isCrashlyticsCollectionEnabled]) {
-    [[Crashlytics sharedInstance] setUserIdentifier:userId];
-  }
-  resolve([NSNull null]);
-}
-
-RCT_EXPORT_METHOD(setUserName:
-  (NSString *) userName
-      resolver:
-      (RCTPromiseResolveBlock) resolve
-      rejecter:
-      (RCTPromiseRejectBlock) reject) {
-  if ([RNFBCrashlyticsInitProvider isCrashlyticsCollectionEnabled]) {
-    [[Crashlytics sharedInstance] setUserName:userName];
-  }
-  resolve([NSNull null]);
-}
-
-RCT_EXPORT_METHOD(setUserEmail:
-  (NSString *) userEmail
-      resolver:
-      (RCTPromiseResolveBlock) resolve
-      rejecter:
-      (RCTPromiseRejectBlock) reject) {
-  if ([RNFBCrashlyticsInitProvider isCrashlyticsCollectionEnabled]) {
-    [[Crashlytics sharedInstance] setUserEmail:userEmail];
+    [[FIRCrashlytics crashlytics] setUserID:userId];
```

Comment:
reading through the SDK docs it does look like it just went away - then this would be something to just include in this PR as release notes or doc updates so people know exactly what APIs they need to migrate and how (otherwise it becomes a support burden with new issues filed...)